packages/devtools_app/test/vm_developer/object_inspector/vm_class_display_test.dart

```diff
@@ -0,0 +1,48 @@
+// Copyright 2022 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'package:devtools_app/devtools_app.dart';
+import 'package:devtools_app/src/screens/vm_developer/vm_class_display.dart';
+import 'package:devtools_test/devtools_test.dart';
+import 'package:flutter/material.dart';
+import 'package:flutter_test/flutter_test.dart';
+import 'package:mockito/mockito.dart';
+
+import '../vm_developer_test_utils.dart';
+
+void main() {
+  final mockClassObject = MockClassObject();
+
+  fakeClass.size = 1024;
+
+  when(mockClassObject.name).thenReturn('FooClass');
+  when(mockClassObject.ref).thenReturn(fakeClassRef);
+  when(mockClassObject.obj).thenReturn(fakeClass);
+  when(mockClassObject.script).thenReturn(fakeScript);
+  when(mockClassObject.instances).thenReturn(fakeInstances);
+  when(mockClassObject.pos).thenReturn(fakePos);
```

Comment:
initialize mockClassObject and perform all this mocking setup inside the `setUp` method.